samples/CosmosDbIntegtation/README.md

```diff
@@ -0,0 +1,65 @@
+# Redis Triggers for Azure Functions
+08/08/2023 Nadia Bugarin, Riley MacDonald, Phoebe Owusu
+
+## Introduction
+This project is an integration between Redis Cache and CosmosDB that includes function triggers using caching patterns that include Write Behind, Write Through, Write Around, and Read Through. Each function trigger, PubSub, Lists, and Streams.
+
+## Prerequisites
+- Accounts
+   - Azure Redis Cache 
+   - CosmosDB Account
+- Visual Studo Code
+    - Custom NuGet Package (available in this repo)
+
+## Description of each caching pattern
+1. WriteThrough: Write-through caching is a data caching strategy that ensures data modifications are immediately written to both the cache and database simultaneously. When data is updated in the system, the write operation I first directed to the cache. Write-through caching ensures that the changes are immediately propagated to the permanent storage.
+    * Benefits: This approach offers strong data consistency, as the cached data and the actual data in the main memory or disk remain in sync at all times. Consequently, even if a system failure or crash occurs, there is minimal risk of data loss or corruption, as the most recent updates have been safely persisted in the permanent storage.
+    * Limitations: Write-through caching might not provide the same level of read performance improvement as write-back caching, which allows for faster read operations by temporarily storing modified data in the cache without immediately writing it to the underlying storage. The trade-off with write-through caching lies in its reliability and data integrity, making it a preferred choice for applications where preserving data consistency is of paramount importance, such as databases and critical file systems.
+    ![Image](Models/WriteThrough.png)
```

Comment:
Adding an extra new line before the image will make it appear cleaner I believe.